dev/com.ibm.ws.org.eclipse.jdt.core/bnd.bnd

```diff
@@ -12,11 +12,11 @@
 bVersion=1.0
 
 Bundle-Name: JDT Compiler
-Bundle-SymbolicName: com.ibm.ws.org.eclipse.jdt.core.3.10.2.v20160712-0000
-Bundle-Description: JDT Compiler, version 4.4.1
+Bundle-SymbolicName: com.ibm.ws.org.eclipse.jdt.core.3.22.0
+Bundle-Description: JDT Compiler, version 3.22.0 [4.16]
```

Comment:
If you are going to put two versions in here it may better to put the date "version" of the Eclipse simultaneous release which includes the 3.22 ECJ: 2020-06.